File: sample/src/main/java/me/xiaopan/sketchsample/bean/BaiduImageSearchResult.java
package me.xiaopan.sketchsample.bean;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class BaiduImageSearchResult {
    @SerializedName("listNum")
    private int total;

    @SerializedName("data")
    private List<BaiduImage> imageList;

    public int getTotal() {
        return total;
    }

    public List<BaiduImage> getImageList() {
        return imageList;
    }
}

File: sample/src/main/java/me/xiaopan/sketchsample/bean/BaiduImageSearchResult.kt
package me.xiaopan.sketchsample.bean

import com.google.gson.annotations.SerializedName

class BaiduImageSearchResult {
    @SerializedName("listNum")
    var total: Int = 0

    @SerializedName("data")
    var imageList: List<BaiduImage>? = null
}
